django_socio_grpc/proto_serializers.py

```diff
@@ -284,6 +297,29 @@ def build_property_field(self, field_name, model_class):
 
         return field_class, field_kwargs
 
+    class _MessageToData(ProtoSerializer._MessageToData):
```

Comment:
add comment to explain the need to inherit _MessageToData for customizing fields that can be nullable